packages/services/api/src/modules/support/providers/support-manager.ts

```diff
@@ -584,6 +586,20 @@ export class SupportManager {
         }),
       );
 
+    await this.auditLog.record({
+      eventType: 'SUPPORT_TICKET_CREATED',
+      ticketDescription: input.description,
+      ticketPriority: input.priority,
+      ticketId: String(response.ticket.id),
+      ticketSubject: input.subject,
+      metadata: {
+        organizationId: input.organizationId,
+        user: currentUser,
+        userEmail: currentUser.email,
+        userId: currentUser.id,
+      },
+    });
+
```

Comment:
```
    await this.auditLog.record({
      eventType: 'SUPPORT_TICKET_CREATED',
      ticketDescription: input.description,
      ticketPriority: input.priority,
      ticketId: String(response.ticket.id),
      ticketSubject: input.subject,
      metadata: {
        organizationId: input.organizationId,
        user: currentUser,
        userEmail: currentUser.email,
        userId: currentUser.id,
      },
    });
```

To me this is metadata (event specific data):
```
      ticketDescription: input.description,
      ticketPriority: input.priority,
      ticketId: String(response.ticket.id),
      ticketSubject: input.subject,
```
and this is not metadata (not stored in metadata column in clickhouse):
```
        organizationId: input.organizationId,
        user: currentUser,
        userEmail: currentUser.email,
        userId: currentUser.id,
```

So it should be

```
    await this.auditLog.record({
      eventType: 'SUPPORT_TICKET_CREATED',
      organizationId: input.organizationId,
      user: currentUser,
      userEmail: currentUser.email,
      userId: currentUser.id,
      metadata: {
        ticketDescription: input.description,
        ticketPriority: input.priority,
        ticketId: String(response.ticket.id),
        ticketSubject: input.subject,
      },
    });
```

Reply:
TODO @TuvalSimha please change this.